svg2svgt/CMakeLists.txt

```diff
@@ -46,9 +48,6 @@ SET(SVG2SVGT_SO_VERSION ${SVG2SVGT_MAJOR})
 ADD_LIBRARY(svg2svgt SHARED ${svg2svgt_SRCS} ${svg2svgt_HDRS} ${svg2svgt_RESOURCES_RCC})
 TARGET_LINK_LIBRARIES(svg2svgt Qt5::Widgets Qt5::Xml Qt5::XmlPatterns)
 SET_TARGET_PROPERTIES(svg2svgt PROPERTIES VERSION ${SVG2SVGT_LIB_VERSION} SOVERSION ${SVG2SVGT_SO_VERSION})
-IF(WIN32)
-    SET_TARGET_PROPERTIES(svg2svgt PROPERTIES SUFFIX "-${SVG2SVGT_SO_VERSION}${CMAKE_SHARED_LIBRARY_SUFFIX}")
-ENDIF(WIN32)
```

Comment:
What's the reason for removing this?

Reply:
This does not work, it makes dll only and no lib. Both are required, so there is an error from linker when cli or gui is built.

I couldn't make this work, so I propose to use new mechanism introduced in CMake 3.4.
These lines are not necessary with new mechanism.

Reply:
I assumed, MSVC is used.
Should I test under MinGW too?

Reply:
Yep that would be good.

Reply:
I've tested with MinGW (MSYS). Your original solution and my change both work. The only difference is that yours builds `libsvg2svgt-0.dll` and mine `libsvg2svgt.dll`.

So, this PR fixes build with MSVC only. Under MinGW fix is not needed.

Reply:
So the name-versioned DLL is intentional. In this case, rather than removing the DLL versioning, please mark the block `IF(MINGW)` ... `ENDIF(MINGW)`.

Reply:
And leave MSVC variant not versioned?

Reply:
So as far as I know there is not really a convention of versioning the dlls in the MSVC world. What error do you actually get if you keep the versioning block and build with MSVC?

Reply:
Missing `svg2svgt.lib`:

```
"U:\svg2svgt\ALL_BUILD.vcxproj" (целевой объект по умолчанию) (1) ->
"U:\svg2svgt\cli\svg2svgt-cli.vcxproj" (целевой объект по умолчанию) (4) ->
(Link конечный объект) ->
  LINK : fatal error LNK1104: эх єфрхЄё  юЄъЁ√Є№ Їрщы "..\svg2svgt\Debug\svg2sv
gt.lib" [U:\svg2svgt\cli\svg2svgt-cli.vcxproj]


"U:\svg2svgt\ALL_BUILD.vcxproj" (целевой объект по умолчанию) (1) ->
"U:\svg2svgt\gui\svg2svgt-gui.vcxproj" (целевой объект по умолчанию) (5) ->
  LINK : fatal error LNK1104: эх єфрхЄё  юЄъЁ√Є№ Їрщы "..\svg2svgt\Debug\svg2sv
gt.lib" [U:\svg2svgt\gui\svg2svgt-gui.vcxproj]

    Предупреждений: 0
    Ошибок: 2
```

(I'm having Russian environment, but I hope you could see the problem)

Reply:
@manisandro, I've returned version suffix for MinGW back